apps/prairielearn/src/lib/question-render.ts

```diff
@@ -630,6 +630,7 @@ export async function renderPanelsForSubmission({
   authorizedEdit,
   renderScorePanels,
   groupRolePermissions,
+  localsOverrides,
```

Comment:
I had an instinct to namespace the `manualGradingInterface` property under some other object, but I've very open to other preferences of how to do this, e.g. just adding `manualGradingInterface` at the top level here.